test/test_core.py

```diff
@@ -9793,16 +9793,23 @@ def test_emscripten_async_load_script(self):
     self.run_process([FILE_PACKAGER, 'test.data', '--preload', 'file1.txt', 'file2.txt', '--from-emcc', '--js-output=script2.js'])
     self.do_runf(test_file('test_emscripten_async_load_script.c'), emcc_args=['-sFORCE_FILESYSTEM'])
 
+  def prep_wsam_worker_in_node(self):
+    # Auto exit after 3 seconds in Nodejs environment to get WASM Worker stdout
```

Comment:
maybe add `TODO: remove this is if/when wasm worker tests exit cleanly under node`

Reply:
this TODO also still needs to be added.

Reply:
Modified as required